src/modules/compliance/src/lib/ComplianceInterface.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "ComplianceInterface.hpp"
+
+// ---------------------------------------------------------------
+//                  MMI module C interface
+// ---------------------------------------------------------------
+
+void ComplianceInitialize(void* log)
+{
+    (void)log;
+}
+
+void ComplianceShutdown(void)
+{
+}
+
+MMI_HANDLE ComplianceMmiOpen(const char* clientName, const unsigned int maxPayloadSizeBytes)
+{
+    (void)clientName;
```

Comment:
>(void)clientName; [](http://example.com/codeflow?start=4&length=17)

We have the UNUSED macro defined in CommonUtils.h:

'''
#define UNUSED(a) (void)(a)
'''